views/index.ejs

```diff
@@ -7,6 +7,7 @@
     <meta name="description" content="Baat Cheet is an anonymous chat app">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title><%= title %></title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css" integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous" />
     <link rel="stylesheet" href="/style.css">
     <link rel="stylesheet" href="/emojify.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
@TanmayBhosale you can remove this.

Reply:
if it is removed the icons do not load

Reply:
Yeah I just realized that. But there are two `cdn` for `font-awesome`, try to remove the other one.

Reply:
done